internal/v1/api.yaml

```diff
@@ -220,6 +220,9 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/BlueprintResponse'
+            application/json;export=true:
```

Comment:
I think that generaly changing response body significantly on change of a header is bit of an anti-pattern, I'm not completely against it, but it smells a bit :)
I'd rather do an edpoint personally.

Reply:
I agree with an explicit endpoint, semantically import/export will be quite different from create/show operations.

Reply:
As an alternative, consider using just URL parameter: `GET blueprint/XXX?export=true` if we document exactly what it does that is something that can be less confusing.